sdk/core/core-client/src/serviceClient.ts

```diff
@@ -25,7 +25,6 @@ import {
 } from "./interfaces";
 import { isStreamOperation } from "./interfaceHelpers";
 import { getRequestUrl } from "./urlHelpers";
-import { isPrimitiveType } from "./utils";
```

Comment:
I think we can actually remove `isPrimitiveType` from utils.ts since this is the only usage of it 😄

Reply:
done